Model/Service/Facet/BuildWebFacetService.php

```diff
@@ -114,6 +114,14 @@ public function getFacets(): Facet
         try {
             $this->populateFilters($includeFilters);
         } catch (Exception $e) {
+            $this->logger->debugWithSource(
```

Comment:
These logging statements seem bloated. The logger erasure should be cleaned up. We have 5-7 lines for one log statement and this is drastically reducing the readability of the code.